### DISCUSSION TITLE
Hiding default cluster navigation

### DISCUSSION THREAD

**Discussion by goyne** [filamentphp/filament#11049] [Help]
### Package

Panel builder

### Package Version

v3.1

### How can we help you?

I have created a cluster holding a resource and multiple pages. Everything works fine, especially the active state of the topbar menu item. 

Now I want to get rid of the cluster's default sub navigation, which is rendered in the left sidebar of the page. Will that be possible? Can't find documentation on that.

Thanks, Jelger

**Comment by mahmoud-757** [comment]
Try  this 
[https://filamentphp.com/docs/3.x/panels/resources/getting-started#resource-sub-navigation](url)

**Comment by leandrocfe** [comment]
You can also add this to each page you want to hide the cluster sub navigation

```php
public function getSubNavigation(): array
{
    return [];
}
```
